### PR TITLE
[DOCS] add options onChange and displayCond to the documentation

### LINKED ISSUE #93: [DOC] it seems the option "displayCond" is not documented
It seems that the option for **displayCond** is not documented inside Content Blocks. I cannot find it. It is available inside the file WithCommonProperties.php.

TYPO3 Docs for this field: https://docs.typo3.org/m/typo3/reference-tca/main/en-us/Columns/Properties/DisplayCond.html

### PR DESCRIPTION
**Pull request by PKuhlmay**
fixes #93

### CODE REVIEW

File: Documentation/YamlReference/FieldTypes/Index.rst [hunk 1]
```diff
@@ -150,6 +150,36 @@ Field options, which can be defined inside the :yaml:`fields` array.
...
+.. confval:: displayCond
+
+   :Required: false
+   :Type: string
+   :Default: ''
+
+   Can be used to display the Content Element only under certain conditions.
+   Please have a look at the `official documentation
+   <https://docs.typo3.org/m/typo3/reference-tca/main/en-us/Columns/Properties/DisplayCond.html#displaycond>`__
+   for more information. **Tip**: When using a condition, the option
```
Comment (nhovratov) [inline Documentation/YamlReference/FieldTypes/Index.rst]: Please use `.. tip::` for the tip. https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Content/Admonitions.html#tip

File: Documentation/YamlReference/FieldTypes/Index.rst [hunk 2]
```diff
@@ -150,6 +150,36 @@ Field options, which can be defined inside the :yaml:`fields` array.
...
 
+.. confval:: displayCond
+
+   :Required: false
+   :Type: string
+   :Default: ''
+
+   Can be used to display the Content Element only under certain conditions.
+   Please have a look at the `official documentation
+   <https://docs.typo3.org/m/typo3/reference-tca/main/en-us/Columns/Properties/DisplayCond.html#displaycond>`__
```
Comment (nhovratov) [inline Documentation/YamlReference/FieldTypes/Index.rst]: Please link this via
```
:ref:`official documentation <t3tca:columns-properties-displaycond>`
```

File: Documentation/YamlReference/FieldTypes/Index.rst [hunk 3]
```diff
@@ -150,6 +150,36 @@ Field options, which can be defined inside the :yaml:`fields` array.
            prefixField: true
            prefixType: vendor
 
+.. confval:: displayCond
+
+   :Required: false
+   :Type: string
```
Comment (garvinhicking) [inline Documentation/YamlReference/FieldTypes/Index.rst]: If I read the code right, it is of mixed type: string|array<string>, which would support multiple conditions?

(I'm working on making a PR to allow converting unprefixed field references to prefixed ones; if this PR is merged I can adjust the docs accordingly after this)

Comment (nhovratov) [reply Documentation/YamlReference/FieldTypes/Index.rst]: Yes string|array. The array version might be a bit difficult to replace.

**Pull request merged**